src/xdotoolify.js

```diff
@@ -298,10 +331,17 @@ _Xdotoolify.prototype.do = async function() {
         commandArr = [];
         await _sleep(op.ms);
       } else if (op.type === 'mousemove') {
+        await this._do(commandArr.join(' '));
```

Comment:
Not quite understanding why this is done up here.

Reply:
It is to wait until everything up to mousemove(which requires ensuring the element with selector exists) is completed. Waiting until the specified timeout happens after that.